### PR TITLE
Replace timestep parameters with frequency to improve readability and robustness

### PR DESCRIPTION
**Pull request by goldenJester**

### CODE REVIEW

File: omnigibson/configs/fetch_behavior.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 env:
-  action_timestep: 0.01666              # (float): environment executes action per action_timestep second
-  physics_timestep: 0.01666             # (float): physics timestep for physx
+  action_frequency: 60                  # (int): environment executes action at the action_frequency rate
+  physics_frequency: 60                 # (int): physics frequency (1 / physics_frequency for physx)
```
Comment (cgokmen) [inline omnigibson/configs/fetch_behavior.yaml:3]: Typo in comment: this is actually 1 / physics timestep

File: omnigibson/configs/tiago_primitives.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 env:
-  action_timestep: 0.01666              # (float): environment executes action per action_timestep second
-  physics_timestep: 0.01666             # (float): physics timestep for physx
+  action_frequency: 60                  # (int): environment executes action at the action_frequency rate
+  physics_frequency: 60                 # (int): physics frequency (1 / physics_frequency for physx)
```
Comment (cgokmen) [inline omnigibson/configs/tiago_primitives.yaml:3]: Typo in comment: this is actually 1 / physics timestep

File: omnigibson/configs/turtlebot_nav.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 env:
-  action_timestep: 0.01666              # (float): environment executes action per action_timestep second
-  physics_timestep: 0.01666             # (float): physics timestep for physx
+  action_frequency: 60                  # (int): environment executes action at the action_frequency rate
+  physics_frequency: 60                 # (int): physics frequency (1 / physics_frequency for physx)
```
Comment (cgokmen) [inline omnigibson/configs/turtlebot_nav.yaml:3]: Typo in comment: this is actually 1 / physics timestep

**Review by cgokmen — CHANGES_REQUESTED** [review]
Also update the PR title to explain what it does (e.g. replace timestep parameters with frequency to improve readability and robustness). Add the link to #360 using the "development" item on the right-side menu.

**Review by cgokmen — APPROVED** [review]

File: omnigibson/envs/env_base.py [hunk 4]
```diff
@@ -186,7 +186,7 @@ def _load_scene(self):
 
         # Set the simulator settings
         # NOTE: This must be done BEFORE the scene is loaded, or else all vision sensors can't retrieve observations
-        og.sim.set_simulation_dt(physics_dt=self.physics_timestep, rendering_dt=self.action_timestep)
+        og.sim.set_simulation_dt(physics_dt=(1 / self.physics_frequency), rendering_dt=(1 / self.action_frequency))
```
Comment (ChengshuLi) [inline omnigibson/envs/env_base.py:189]: should we in general do `1.0 / frequency` to be 100% safe? I know probably no one will use this old a version of python...

**Review by ChengshuLi — APPROVED** [review]

**Pull request merged**